### PR TITLE
Add surface heat fluxes to ocean climatology tasks

### PR DESCRIPTION
**Pull request by cbegeman**
Add surface heat fluxes to ocean climatology tasks

### CODE REVIEW

**Comment by xylar** [comment]
@cbegeman, https://github.com/MPAS-Dev/MPAS-Analysis/pull/1011 tuned out to be a mistake.  I'll revert that as soon as I have time.  In the meantime, maybe rebase this to exclused that change.

**Comment by xylar** [comment]
> @cbegeman, https://github.com/MPAS-Dev/MPAS-Analysis/pull/1011 tuned out to be a mistake. I'll revert that as soon as I have time. In the meantime, maybe rebase this to exclude that change.

This issue has been resolved on conda-forge and #1011 should now work fine.  No need to rebase after all.

**Comment by cbegeman** [comment]
@xylar Thanks! I haven't yet been able to figure out why the heat and mass fluxes are ending up in the same subsection. I might be interpreting some of the gallery/group arguments to `set_plot_info` incorrectly. If you have time to take a look, that could be helpful.

https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.cbegeman/analysis/20240626.HRr5-test11.chrysalis/climo_section_yrs1-10_latlon/ocean/index.html#global_ocnfluxclimo

**Comment by xylar** [comment]
@cbegeman, I think the answer is just that there are only 2 levels of hierarchy -- gallery groups and galleries.  Gallery groups also have a subtitle but that can't be different for different gallery groups.  The're just additional information about the gallery group.

**Comment by cbegeman** [comment]
@xylar That's helpful. Thanks! I'll fix that up and update the docs and then I'll mark it ready for review.

**Review by xylar — APPROVED** [review]
I ran the test suite on Chrysalis.  Results from the tests are here:
https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.xasay-davis/analysis_testing/chrysalis/enhance-ocean-climo-flux-task/

@cbegeman, another really great addition!

File: mpas_analysis/ocean/climatology_map_fluxes.py [hunk 1]
```diff
@@ -45,94 +44,170 @@ def __init__(self, config, mpasClimatologyTask, controlConfig=None):
...
+                            self).customize_masked_climatology(climatology,
+                                                               season)
+
+        # calculate heat flux from temperature flux
+        scaleFactor = constants.rho_sw * constants.cp_sw  # C m s^-1 to W m^-2
+        heatFlux = 0.5 * scaleFactor * climatology[variable]
+        # drop unnecessary fields before re-mapping
+        climatology.drop_vars([variable])
+
+        # this creates a new variable eke in climatology (like netcdf)
```
[the remark this answers is not in the document]
Comment (cbegeman) [reply mpas_analysis/ocean/climatology_map_fluxes.py:206]: Thanks for catching that! And thanks for reviewing and testing! I'll fix and then merge

Comment (cbegeman) [inline mpas_analysis/ocean/climatology_map_fluxes.py:206]:
```suggestion
        # this creates a variable with heat flux units in climatology (like netcdf)
```

**Pull request merged**